test.py

```diff
@@ -0,0 +1,65 @@
+import os
+import pytest
+import subprocess
+import testinfra
+
+@pytest.fixture(scope='session')
+def host(request):
+    subprocess.check_call(['docker', 'build', '-t', 'radicale', '.'])
+    docker_id = subprocess.check_output(['docker', 'run', '-d', 'radicale']).decode().strip()
+    
+    yield testinfra.get_host("docker://" + docker_id)
+    
+    # teardown
+    subprocess.check_call(['docker', 'rm', '-f', docker_id])
+
+def test_system(host):
+    assert host.system_info.distribution == 'alpine'
+    assert host.system_info.release == '3.7.0'
+
+def test_entrypoint(host):
```

Comment:
Same here, I would skip this test (see main comment)

Reply:
I'll drop it. As you mention the entrypoint is just a means to running radicale, so from a black box perspective it doesn't really matter that it exists, just that radicale starts up